.editorconfig

```diff
@@ -0,0 +1,57 @@
+# To learn more about .editorconfig see https://aka.ms/editorconfigdocs
+
+###############################
+# Core EditorConfig Options   #
+###############################
+# All files
+[*]
+end_of_line              = crlf
+indent_style             = space
+insert_final_newline     = true
+trim_trailing_whitespace = true
+# Code files
+[*.{cs,csx,fs,fsx,vb,vbx}]
+indent_size          = 4
+charset              = utf-8-bom
+# Project files
+[*.{csproj,fsproj,vbproj,shproj,projitems}]
+indent_size          = 2
+charset              = utf-8-bom
+
+###############################
+# F# Coding Conventions       #
+###############################
+[*.{fs,fsx}]
+max_line_length = 120
```

Comment:
Please don't add all the possible Fantomas settings.
This is not recommended, https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#configuration

> Please note that you should only add settings to the .editorconfig file when you want to deviate from the default settings. Copying the entire list above is unnecessary.

Reply:
Thank you for the suggestion!